lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -650,6 +671,15 @@ def test_cache_map_bad_dir_path(
                 0,
                 CacheType.FILE,
             ),
+            (
+                "dir_name/subdir1/f12_sym",
+                "dir_name/subdir1/f12_sym",
+                "f12_sym",
+                Path("dir_name"),
+                CacheType.DIRECTORY,
+                None,
+                CacheType.SYMLINK,
+            ),
```

Comment:
Apparently I'm suffering from a personal "off by one" error....

If we were to look at the list of files in the theoretical tarball whose unpack area is produced by `generate_test_result_tree()` the first file path would be `dir_name/`, right?

My intention for `f12_sym` was for it to refer to the _parent_ of the root of the unpack area (that is, it should contain a _relative_ path to an _existing_ file _outside_ the result area).

So, I guess the link contents should be `../..`; then I believe that `resolve_path` would be `Path("../..")` and `resolve_type` would be `CacheType.SYMLINK` (or `CacheType.OTHER` if you accept my earlier suggestion), because it's referencing something outside the results area.

Reply:
My recommendation for `f12_sym` is that it be a symlink with a relative path referencing an existing file/directory which is _outside_ the result tree, so I think this definition should be something like this:
```python
            (
                "dir_name/subdir1/f12_sym",
                "dir_name/subdir1/f12_sym",
                "f12_sym",
                Path("../.."),
                CacheType.OTHER,
                None,
                CacheType.SYMLINK,
            ),
```